SayTheirNames/Source/Controller/More/MoreController.swift

```diff
@@ -24,16 +24,19 @@
 
 import UIKit
 
-class AboutController: UIViewController {
-    private let aboutView = AboutView()
+class MoreController: UIViewController {
 
     required init() {
         super.init(nibName: nil, bundle: nil)
     }
     
     required init?(coder: NSCoder) { fatalError("This should not be called") }
-
-    override func loadView() {
-        self.view = aboutView
+    
```

Comment:
If we use `loadView` we dont have to have our UI as a subview
```suggestion
    override func loadView() {
    view = moreView
```